packages/manager/modules/telecom-dashboard/src/identity-check/form/telecom-dashboard-identity-check-form.controller.js

```diff
@@ -4,6 +4,8 @@ import confirmTemplate from './confirm/confirm.html';
 export default class IdentityCheckFormCtrl {
   /* @ngInject */
   constructor(
+    $state,
```

Comment:
nitpick(blocking): Avoid using $state in controller files.
You can move the $state.go to routing.js